Lib/test/test_pdb.py

```diff
@@ -363,6 +363,42 @@ def test_pdb_breakpoint_commands():
     4
     """
 
+def test_pdb_breakpoint_on_annotated_function_def():
```

Comment:
Are there tests for functions with decorators?

Reply:
Not really. What's the expected behavior? We can't resolve the decorator in pdb, we will only add the breakpoint inside the decorated function.

Reply:
Well, that's not true.. It depends on whether the function is in the name space. If it's already in the namespace, we will set the breakpoint in the decorator.

Reply:
I don't know what the expected behaviour would be. Still interesting what the behaviour is.

Reply:
So if it's not compiled, we will be searching based on source code, and we will break inside the function. If it is compiled, then the name in the name space will point to a decorated function, and the code object would be the return value of the decorator, so probably a wrapper.